stl/inc/mutex

```diff
@@ -536,6 +541,7 @@ struct _Init_once_completer {
     ~_Init_once_completer() {
         if (_RENAME_WINDOWS_API(__std_init_once_complete)(&_Once._Opaque, _DwFlags, nullptr) == 0) {
             _CSTD abort();
+            _LINK_ALTERNATE_NAMES; // never called, forces xonce2.cpp to be linked

```

Comment:
The compiler is smart enough to throw away unreachable code: https://godbolt.org/z/c7cv8vd8z

And if it wasn't I'd objected this, as it bloats fast path code. Think of a better way to prevent throwing away `.cpp`. Maybe linker options or something.

Reply:
This is optimized away by the compiler but still enforces linking the object in.